core/src/main/java/org/elasticsearch/ingest/IngestModule.java

```diff
@@ -81,4 +83,10 @@ public void registerProcessor(String type, BiFunction<Environment, TemplateServi
     public static boolean isIngestEnabled(Settings settings) {
         return settings.getAsBoolean("node.ingest", true);
     }
+
+    public static boolean isIngestEnabled(ImmutableOpenMap<String, String> nodeAttributes) {
```

Comment:
maybe this static helper should be in `IngestProxyActionFilter`? It doesn't seem to be used elsewhere.